srcpkgs/tuner/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'tuner'
+pkgname=tuner
+version=1.5.1
+revision=1
+build_style=meson
+hostmakedepends="glib-devel gettext pkg-config vala"
+makedepends="geoclue2-devel geocode-glib-devel geocode-glib-libsoup2-devel
```

Comment:
you are mixing libsoup2 and libsoup3 versions of geocode-glib here, remove the libsoup3 one (geocode-glib-devel) or switch the whole package to libsoup3 if possible.

Reply:
Is this OK?

Reply:
Yes, do you know if there is a plan to support libsoup3?

Reply:
> Yes, do you know if there is a plan to support libsoup3?

No, I'm not in contact with the author.